src/components/PermissionsManagement/PermissionListItem.jsx

```diff
@@ -153,7 +153,7 @@ function PermissionListItem(props) {
                 props.onChange();
               }}
               // eslint-disable-next-line react/destructuring-assignment
-              disabled={!props.hasPermission('putRole')}
+              disabled={!(props.hasPermission('putRole') || props.hasPermission('postRole'))}
```

Comment:
```suggestion
              disabled={!(props.isEditing ? props.hasPermission('putRole') : props.hasPermission('postRole'))}
```
Your changes would allow users with 'postRole' to also edit existing roles. Would need to do something like the suggestion where you check if the role is new to use the corresponding permission (put vs post).

Reply:
Hey Nathan, thanks for your review and suggestion! Following your logic, I have updated my code. Please have a check when you have time!

Reply:
Just noticed I got a failed npm test. And I found that the failed case is from other part of the codebase. Could you also have a check to confirm it? Thanks!! @nathanah